test-node.bash

```diff
@@ -110,8 +118,21 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 
-if $force_init; then
+if $dev_build; then
+  if $force_init; then
     force_build=true
+  fi
+  if [[ "$(docker images -q nitro-node-dev:latest 2> /dev/null)" == "" ]]; then
+    force_build=true
+  fi
+  if [[ "$(docker images -q blockscout:latest 2> /dev/null)" == "" ]]; then
+    force_build=true
+  fi
+else
+  if $force_build; then
+    echo "Can only build in dev mode with --dev"
```

Comment:
without --dev, you probably still want to build testnode-scripts (incl. on --init)